pandas/core/generic.py

```diff
@@ -155,7 +155,13 @@ class NDFrame(PandasObject, SelectionMixin):
     )  # type: FrozenSet[str]
     _metadata = []  # type: List[str]
     _is_copy = None
-    _data = None  # type: BlockManager
+    _mgr = None  # type: BlockManager
+
+    @property
```

Comment:
is this actually hit anywhere? (maybe in pyarrow)?

Reply:
It's not hit in our tests, but since pyarrow/fastparquet still reaches into core.internals, I figured we needed the alias for compat